VideoWeb/VideoWeb/ClientApp/src/assets/i18n/en.json

```diff
@@ -382,7 +382,8 @@
   "judge-context-menu": {
     "context-menu-icon": "Context menu icon",
     "call-witness": "Call witness",
-    "dismiss-witness": "Dismiss witness",
+    "admit-participant": "Admit {{ role }}",
```

Comment:
Ticket VIH-8253 pending for Welsh

Reply:
I am missing this part wow the role gets inserted here

Reply:
I can show you how this works. I think I have simpler examples somewhere, the one here ends up going though multiple pipes.